addons/bus/static/src/services/bus_service.js

```diff
@@ -49,11 +49,6 @@ export const busService = {
         const bus = new EventBus();
         const notificationBus = new EventBus();
         const subscribeFnToWrapper = new Map();
-        /**
-         * @typedef {typeof import("@bus/workers/websocket_worker").WORKER_STATE} WORKER_STATE
-         * @type {WORKER_STATE[keyof WORKER_STATE]}
-         */
-        let isInitialized = false;
```

Comment:
Note: not needed anymore. The only occurences were used for the following condition: `!connectionInitializedDeferred && !isInitialized`. However, there is no way for `isInitialized`  to be true without `connectionInitializedDeferred` being truthy as well. It was previously used in the worker handling logic, which now belongs to the worker service.